src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderEmploymentTransformer.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.hmpps.prison.service.transformers;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+import uk.gov.justice.hmpps.prison.api.model.OffenderEmploymentResponse;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderEmployment;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderEmployment.EmploymentPostType;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderEmployment.PayPeriodType;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderEmployment.ScheduleType;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.ReferenceCode;
+import uk.gov.justice.hmpps.prison.service.AddressTransformer;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+@Component
+public class OffenderEmploymentTransformer implements Converter<OffenderEmployment, OffenderEmploymentResponse> {
+    @Override
+    public OffenderEmploymentResponse convert(@NotNull final OffenderEmployment offenderEmployment) {
+        return new OffenderEmploymentResponse(
+            offenderEmployment.getId().getBookingId(),
```

Comment:
how do you know you've got the fields here in the correct order - we don't tend to use constructors for this many arguments - should be a builder